config/manager/patches/image.yaml

```diff
@@ -148,6 +148,10 @@ spec:
               value: cockroachdb/cockroach:v21.2.13
             - name: RELATED_IMAGE_COCKROACH_v21_2_14
               value: cockroachdb/cockroach:v21.2.14
+            - name: RELATED_IMAGE_COCKROACH_v21_2_15
```

Comment:
Hmm, looks like we need to merge something into master?

Reply:
Done.